OWML.ModHelper.Input/ModCommandListener.cs

```diff
@@ -75,7 +97,18 @@ private void Update()
                 {
                     OnTapped?.Invoke(command);
                 }
+                if (blockFlag)//damn you, Mobius Digital
+                {
+                    if (!(_wasPressed[command] || _isPressed[command]))
+                    {
+                        command.SetValue("_blockNextRelease", false);
```

Comment:
var toBlock = ...;
command.SetValue(..., toBlock);

Reply:
(this removes the if)

Reply:
done